packages/appstore/src/components/main-title-bar/asset-summary.tsx

```diff
@@ -1,112 +1,43 @@
-            getCurrentExchangeRate(cfd_real_balance.currency, setExchangedRateCfdReal);
-        }
-        if (cfd_demo_balance.currency !== platform_demo_balance.currency) {
-            getCurrentExchangeRate(cfd_demo_balance.currency, setExchangedRateCfdDemo, platform_demo_balance.currency);
-        }
-    }, [
-        cfd_demo_balance.currency,
-        cfd_real_balance.currency,
-        getExchangeRate,
-        platform_demo_balance.currency,
-        platform_real_balance.currency,
-    ]);
-
-    const getTotalBalance = () => {
-        if (selected_account_type === 'real') {
-            return {
-                balance: platform_real_balance.balance + cfd_real_balance.balance * exchanged_rate_cfd_real,
-                currency: platform_real_balance.currency,
-            };
-        }
-
-        return {
-            balance: platform_demo_balance.balance + cfd_demo_balance.balance * exchanged_rate_cfd_demo,
-            currency: platform_demo_balance.currency,
-        };
-    };
-
-    const has_active_related_deriv_account = !((no_CR_account && !is_eu_user) || (no_MF_account && is_eu_user)); // if selected region is non-eu, check active cr accounts, if selected region is eu- check active mf accounts
-    const eu_account = is_eu_user && !no_MF_account;
-    const cr_account = !is_eu_user && !no_CR_account;
-
-    //dont show loader if user has no respective regional account
-    if ((is_switching || is_logging_in) && (eu_account || cr_account)) {
-        return (
-            <React.Fragment>
-                <div className='asset-summary__container content-loader'>
-                    <TotalAssetsLoader />
-                </div>
-            </React.Fragment>
-        );
-    }
+    if (!(has_active_related_deriv_account || selected_account_type === 'demo'))
+        return <div className='asset-summary' />;
 
     return (
         <div className='asset-summary'>
-            {has_active_related_deriv_account || selected_account_type === 'demo' ? (
-                <React.Fragment>
-                    {!isMobile() ? (
-                        <Text align='right' size='xs' line_height='s'>
-                            {localize('Total assets')}
-                        </Text>
-                    ) : null}
-                    <Popover
-                        alignment={isMobile() ? 'top' : 'left'}
-                        message={localize('Total assets in all your accounts')}
-                        zIndex={9999}
-                        is_bubble_hover_enabled
-                    >
-                        <BalanceText
-                            currency={getTotalBalance().currency}
-                            balance={getTotalBalance().balance}
-                            underline_style='dotted'
-                        />
-                    </Popover>
-                </React.Fragment>
-            ) : null}
+            <React.Fragment>
```

Comment:
@farzin-deriv I think we don't need this <React.Fragment> here because it's inside div